.github/workflows/impress.yml

```diff
@@ -120,29 +106,49 @@ jobs:
       DB_NAME: impress
       DB_USER: dinum
       DB_PASSWORD: pass
-      DB_PORT: 5432
+      DB_PORT: 15432
+      STORAGES_STATICFILES_BACKEND: django.contrib.staticfiles.storage.StaticFilesStorage
+      AWS_S3_ENDPOINT_URL: http://localhost:9000
+      AWS_S3_ACCESS_KEY_ID: impress
+      AWS_S3_SECRET_ACCESS_KEY: password
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
-      - name: Create writable /data
+        uses: actions/checkout@v4
+
+      - name: Setup base
+        run: |
+          make data/media
+          make create-env-files
+
+      - name: Run containers
         run: |
-          sudo mkdir -p /data/media && \
-          sudo mkdir -p /data/static
+          DOCKER_USER=$(id -u):$(id -g)
+          export DOCKER_USER
+          docker compose up --force-recreate -d postgresql
+          docker compose up --force-recreate -d createbuckets
```

Comment:
I don't think we want to run docker-compose here

Reply:
Other way: [👷(CI) configure Minio with the CI](https://github.com/numerique-gouv/impress/pull/27/commits/f054b07f2225ecaefa2c471c2c0f17cf0eff12a8)

Reply:
The official image of Minio cannot be started from github actions **services**, these discussions talk about it:
https://stackoverflow.com/questions/64031598/creating-a-minios3-container-inside-a-github-actions-yml-file
https://stackoverflow.com/questions/60849745/how-can-i-run-a-command-in-github-action-service-containers